osier/tech_library.py

```diff
@@ -0,0 +1,180 @@
+                                    lifecycle_co2_rate=1.3e-4*co2_eq_units,
+                                    land_intensity=0.0,
+                                    )
+coal = ThermalTechnology(technology_name='Coal_Conv',
+                         capacity=0*GW,
+                         capital_cost=1000*(1/kW)*to_MDOLLARS,
+                         om_cost_variable=0.0*to_MDOLLARS,
+                         om_cost_fixed=40.7033*(1/kW)*to_MDOLLARS,
+                         fuel_cost=21.369*(1/(MW*hour))*to_MDOLLARS,
+                         ramp_up_rate=0.5,
+                         ramp_down_rate=0.5,
+                         lifecycle_co2_rate=1e-3*co2_eq_units,
+                         land_intensity=0.0,
+                         )
+coal_adv = ThermalTechnology(technology_name='Coal_Adv',
+                            capacity=0*GW,
+                            capital_cost=4924.6*(1/kW)*to_MDOLLARS,
+                            om_cost_variable=0.0*to_MDOLLARS,
+                            om_cost_fixed=58.24*(1/kW)*to_MDOLLARS,
+                            fuel_cost=36.6329*(1/(MW*hour))*to_MDOLLARS,
+                            ramp_up_rate=0.5,
+                            ramp_down_rate=0.5,
+                            lifecycle_co2_rate=3.7e-4*co2_eq_units,
+                            land_intensity=0.0,
+                            )
+battery = StorageTechnology(technology_name='Battery',
+                            capacity=0.81534126*GW,
+                            capital_cost=613*(1/kW)*to_MDOLLARS,
+                            om_cost_variable=0*to_MDOLLARS,
+                            om_cost_fixed=15.32*(1/kW)*to_MDOLLARS,
+                            fuel_cost=0*to_MDOLLARS,
+                            storage_duration=4,
+                            efficiency=0.85,
+                            initial_storage=0,
+                            capacity_credit=0.5,
+                            lifecycle_co2_rate=3.3e-5*co2_eq_units,
+                            land_intensity=0.0,
+                            )
+wind = Technology(technology_name='WindTurbine',
+                  renewable=True,
+                  dispatchable=False,
+                  fuel_type='wind',
+                  capacity=0*GW, 
+                  capital_cost=1180.6*(1/kW)*to_MDOLLARS,
+                  om_cost_fixed=33.11*(1/kW)*to_MDOLLARS,
+                  lifecycle_co2_rate=1.2e-5*co2_eq_units,
+                  land_intensity=0.0,
+                  capacity_credit=0.35)
+solar = Technology(technology_name='SolarPanel',
+                  renewable=True,
+                  dispatchable=False,
+                  fuel_type='solar',
+                  capacity=2.8103015*GW, 
+                  capital_cost=673.2*(1/kW)*to_MDOLLARS,
+                  om_cost_fixed=8.05*(1/kW)*to_MDOLLARS,
+                  lifecycle_co2_rate=3.7e-5*co2_eq_units,
+                  land_intensity=0.0,
+                  capacity_credit=0.19)
+
```

Comment:
I think it would be really helpful to have the sources for all this data in the docs for Osier

Reply:
That is coming later :)